https://issues.redhat.com/browse/ACM-19867 for 2.14 (#8138)
open-cluster-management/rhacm-docs: governance/supported_policies.adoc

| Policy sample | Description

| Namespace policy

> | Ensure consistent environment isolation and naming with Namespaces.

Author: ```suggestion
| Ensure consistent environment isolation and naming with namespaces.
```

| Pod policy
| Ensure cluster workload configuration.

| Memory usage policy

> | Limit workload resource usage using Limit Ranges.

Author: ```suggestion
| Limit workload resource usage using limit ranges.
```

| Image vulnerability policy
| Deploy the Container Security Operator and detect known image vulnerabilities in pods running on the cluster.

| {gate} deployment
| Gatekeeper is an admission webhook that enforces custom resource definition-based policies that are run by the Open Policy Agent (OPA) policy engine. The {gate} is available for installing Gatekeeper.

| Gatekeeper compliance policy
| After deploying Gatekeeper to the clusters, deploy this sample Gatekeeper policy that ensures namespaces that are created on the cluster are labeled as specified.

| Red Hat OpenShift Platform Plus policy set
| Red Hat OpenShift Platform Plus is a hybrid-cloud suite of products to securely build, deploy, run, and manage applications for multiple infrastructures. You can deploy Red Hat OpenShift Platform Plus to managed clusters using `PolicySets` delivered through a {acm-short} application. 
|===

{ocp} 4.x also supports the {acm-short} configuration policies.

[#additional-resources-supp-pol]
== Additional resources

View the following policy documentation to learn how to apply policies:

> * See xref:../governance/supported_policies.adoc#supported-policies[Supported {acm} policies]

Author: ```suggestion
* To learn about the supported policies, see xref:../governance/supported_policies.adoc#supported-policies[Supported {acm} policies].
```

> * See the Kubernetes link:https://kubernetes.io/docs/reference/access-authn-authz/rbac/[RBAC documentation].

Author: ```suggestion
* Find more details about Kubernetes role-based access control in the Kubernetes link:https://kubernetes.io/docs/reference/access-authn-authz/rbac/[RBAC documentation].
```

> * See link:https://docs.redhat.com/documentation/en-us/openshift_container_platform/4.17/html/authentication_and_authorization/managing-pod-security-policies[Managing Security Context Constraints documentation] in the {ocp-short} documentation.

Author: ```suggestion
* Review the {ocp-short} documentation for more details about link:https://docs.redhat.com/documentation/en-us/openshift_container_platform/4.17/html/authentication_and_authorization/managing-pod-security-policies[Managing Security Context Constraints documentation].
```

> * See link:https://docs.redhat.com/documentation/en-us/openshift_container_platform/4.17/html/security_and_compliance/encrypting-etcd[Encrypting etcd data] in the {ocp-short} documentation.

Author: ```suggestion
* Review the {ocp-short} documentation for more information about link:https://docs.redhat.com/documentation/en-us/openshift_container_platform/4.17/html/security_and_compliance/encrypting-etcd[Encrypting etcd data].
```

* See link:https://docs.redhat.com/documentation/en-us/openshift_container_platform/4.17/html/security_and_compliance/compliance-operator#understanding-compliance-operator[Understanding the Compliance Operator] in the {ocp-short} documentation.

> * See the link:https://github.com/quay/container-security-operator#readme[Container Security Operator] GitHub repository.

Author: ```suggestion
* For more details about the Container Security Operator, see the link:https://github.com/quay/container-security-operator#readme[Quay GitHub repository].
```

> * See the xref:../governance/gatekeeper_operator/gk_operator_overview.adoc#gk-operator-overview[{gate} overview].

Author: ```suggestion
* Learn how you can use Gatekeeper with {acm-short} by reading the xref:../governance/gatekeeper_operator/gk_operator_overview.adoc#gk-operator-overview[{gate} overview].
```

> * See the link:https://open-policy-agent.github.io/gatekeeper/website/docs/[Gatekeeper] documentation for more details.

Author: ```suggestion
* To learn more about what policy libraries are offered or just overall functionality of Gatekeeper, see the link:https://open-policy-agent.github.io/gatekeeper/website/docs/[Gatekeeper documentation].
```

* See the {ocp-short} documentation for link:https://docs.redhat.com/en/documentation/openshift_platform_plus/4[OpenShift Platform Plus].

> //add link to template processing to the Governance intro topic

Author: ```suggestion
```

> * See xref:../governance/grc_intro.adoc#governance[Governance] for more details.
> 
> * For details about running Ansible jobs on violated policies, see xref:../governance/ansible_grc.adoc#configuring-governance-ansible[Configuring {aap-short} for Governance].

Author: ```suggestion
* For details about running Ansible jobs on violated policies, see xref:../governance/ansible_grc.adoc#configuring-governance-ansible[Configuring {aap-short} for Governance].

* See xref:../governance/grc_intro.adoc#governance[Governance] for more details about the feature.
```

| Compliance operator CIS scan
| After applying the Compliance operator policy, deploy a Center for Internet Security (CIS) scan to check for compliance with CIS security profiles.

| Image vulnerability policy
| Deploy the Container Security Operator and detect known image vulnerabilities in pods running on the cluster.

| {gate} deployment
| Gatekeeper is an admission webhook that enforces custom resource definition-based policies that are run by the Open Policy Agent (OPA) policy engine. The {gate} is available for installing Gatekeeper.

| Gatekeeper compliance policy
| After deploying Gatekeeper to the clusters, deploy this sample Gatekeeper policy that ensures namespaces that are created on the cluster are labeled as specified.

| Red Hat OpenShift Platform Plus policy set
| Red Hat OpenShift Platform Plus is a hybrid-cloud suite of products to securely build, deploy, run, and manage applications for multiple infrastructures. You can deploy Red Hat OpenShift Platform Plus to managed clusters using `PolicySets` delivered through a {acm-short} application. 
|===

{ocp} 4.x also supports the {acm-short} configuration policies.

> 

Reviewer: Thanks for taking this--it has been on the books for a while. Looks good on the table! 

The only issue I see is the  long list of links under the table. We need another plan for those, I think? First I need to understand why we need 20+ lines of links, then we can better set this up. Things to consider:

- Create the header for additional info, like we have in other topics. (this will be "related links" in DITA)

- It seems that all these links are to guide the user for something, but this is in a topic titled Support policies.

- Let's also make sure none of these are deep links because we have had to fix a lot of deeper links in GRC. We should link to the highest topic in most cases.

- We should not need to point to Kubernetes namespace doc, that is a kew component of Kube and is used in all the components and we don't point to it. I see some of these are more specific.

- Also consider just pointing to the intro of the Governance doc, because it seems we are linking to a big part of it anyway lol.

- I am out tomorrow. I am happy to discuss Monday, but I would rather not merge without a bit better plan for the links.

Author: Thanks Brandi, I appreciate your thorough review. Great catch, I didn't realize that I didn't have the "Additional resources" section. Sho provided some links to our own doc and was suggesting to add the links to help users navigate to specific areas. I took a look through our doc a learned that I was the only one using inline links within tables. So I moved the links out of the table.

Reviewer: Yeah, we should not have them inside the table.

Reviewer: It looks better, but we have some with punctuation and some without:

Additional resources
View the following policy documentation to learn **more about**....:

The let's let them know _why_ they should go to the link, like you do with the Ansible link.

* See ___ for information about support.
* For Kubernetes policy information, see....
* Learn about Gatekeeper....

(Something like that, you don't have to start the whole list with "see" and make sure you punctuate. Then I think we are good on this one.)
